core-ui/src/components/Extensibility/plugins/VisibilityHandler.js

```diff
@@ -20,6 +20,10 @@ export function VisibilityHandler({
 
   // rule.visibility won't work for "var", we must use schema.get('visibility')
   const visibilityFormula = schema.get('visibility');
+  const overwrite =
+    typeof schema.get('overwrite') !== 'undefined'
+      ? schema.get('overwrite')
+      : true;
```

Comment:
wouldn't `??` be enough here?
```suggestion
  const overwrite = schema.get('overwrite') ?? true;
```